pl_bolts/datamodules/kitti_datamodule.py

```diff
@@ -21,6 +21,7 @@ def __init__(
             num_workers: int = 16,
             batch_size: int = 32,
             seed: int = 42,
+            shuffle: bool = False,
```

Comment:
Shall we have `drop_last` and `pin_memory` here like other datamodules?

Reply:
Yeah I'm on it!